src/graphql/queries/ListReplies.js

```diff
@@ -83,7 +83,17 @@ export default {
       shouldQueries.push(
         {
           more_like_this: {
-            fields: ['text', 'reference'],
+            fields: ['text'],
```

Comment:
What would happen if we use the original setup?

Reply:
Let's see the test case `ListReplies filters by moreLikeThis and given text, find replies with the said URL's content`:
The original fixture `moreLikeThis2`(reference:barbar) should match, but it only matches `userFoo` and `rumor`.

Reply:
I see, very interesting. Seems that using `fields: ['text', 'reference']` would concat the two field into one, which may cause `not being selected as representative terms [in `more_like_this` process](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-mlt-query.html#_how_it_works). Splitting into separate more_like_this should indeed fix this.